indico/modules/events/surveys/templates/survey_announcement.html

```diff
@@ -0,0 +1,28 @@
+{% set survey_count = surveys|length %}
```

Comment:
Writing `|length` twice is not such a big deal. DRY is a good thing, but I believe saving a single cheap function call isn't a good argument for it. The point of DRY is more to avoid repeating bigger pieces of code (things where you'd copy&paste, not just type a few characters twice)